_security/access-control/users-roles.md

```diff
@@ -242,3 +242,42 @@ Map the role to your user:
 1. Choose the **Mapped users** tab and **Manage mapping**.
 1. For **Internal users**, add your bulk access user.
 1. Choose **Map**.
+
+## Admin and super admin roles in OpenSearch security
+
+User roles in OpenSearch are essential for controlling access to cluster resources and functionalities. Users can be categorized as regular users, admin users, or super admin users based on their access rights and responsibilities.
+
+For more information about defining a user, see [defining users](https://opensearch.org/docs/latest/security/access-control/users-roles/#defining-users). For more information about defining roles, see [defining roles](https://opensearch.org/docs/latest/security/access-control/users-roles/#defining-roles).
+
+
+### Regular users
+Regular users have basic access permissions to interact with the OpenSearch cluster, such as querying data and using dashboards, but they do not have administrative privileges.
+
+### Admin users
+Admin users in OpenSearch have elevated permissions allowing them to perform various administrative tasks within the cluster. They have broader access compared to regular users, which includes:
+- Managing users and roles
+- Configuring permissions
+- Adjusting backend settings
+
+Admin users can perform these tasks by configuring settings in the `opensearch.yml` file, in OpenSearch Dashboards, or with the REST API. For more information about configuring users and roles, see [predefined roles](https://opensearch.org/docs/latest/security/access-control/users-roles/#predefined-roles).
+
+### Super admin users
+Super admins hold the highest level of administrative authority within the OpenSearch environment. This role is typically reserved for select users and is maintained carefully.
+
+Super admin users have unrestricted access to all settings and data within the cluster, including the following tasks:
+- Modify Security plugin configurations
+- Access and manage the security index `.opendistro_security`
+- Override any security limitations
+
+#### Authentication of super admin role
+
+Super admins are authenticated through certificates, not passwords. The necessary certificates are defined in the `admin_dn` section of the `opensearch.yml` file and must be signed with the same root CA to verify and connect it to the cluster.
+```
+YAML
+plugins.security.authcz.admin_dn:
+- CN=kirk,OU=client,O=client,L=test, C=de
+``` 
+
+If the super admin certificate is signed by a different CA, then the admin CA must be concatenated with the node's CA in the file defined in `plugins.security.ssl.http.pemtrustedcas_filepath` in `opensearch.yml`. 
+
+For more information, see [Configuring super admin certificates](https://opensearch.org/docs/latest/security/configuration/tls/#configuring-super-admin-certificates).
```

Comment:
Do we need to update the section here to say "super admin" instead of "admin"?: https://opensearch.org/docs/latest/security/configuration/tls/#configuring-admin-certificates

Reply:
Yes thanks @hdhalter done!